src/components/20-molecules/policy-features/policy-features-item/ui.test.js

```diff
@@ -0,0 +1,115 @@
+import { Selector } from 'testcafe';
+
+const host = process.env.TEST_HOST_STORYBOOK_URL || 'http://localhost:9999';
+const TAG = 'axa-policy-features-item';
+const CLASS = '.policy-features-item';
+const defaultWindowHeight = 1000;
+const lgWindowWidth = 992;
+const mdWindowWidth = 768;
+const smWindowWidth = 576;
+
+// Selectors
+const firstSvgElementInDocument = Selector(
+  () => document.querySelector('axa-policy-features-item').shadowRoot
+).find('svg');
+const firstH1ElementInDocument = Selector(
```

Comment:
I would love to have a better semantic here. What exacly is the first h1 element? I guess you want to test the title of the first item or whatever, or the title of the policy feature. I like to name my variables around the content they contain as value.